packages/frontend/app/routes/wallet-addresses._index.tsx

```diff
@@ -38,6 +39,11 @@ export default function WalletAddressesPage() {
     useLoaderData<typeof loader>()
   const navigate = useNavigate()
 
+  const badgeColorByStatus: Record<WalletAddressStatus, BadgeColor> = {
+    [WalletAddressStatus.Active]: BadgeColor.Green,
+    [WalletAddressStatus.Inactive]: BadgeColor.Red
+  }
```

Comment:
This can be moved outside the component to avoid creating a new object every time the page is rendered.